packages/@react-spectrum/picker/docs/Picker.mdx

```diff
@@ -266,6 +266,43 @@ Items within Picker also allow for additional content used to better communicate
 </Picker>
 ```
 
+## Asynchronous loading
```

Comment:
In which order should we put this section? I just put it at the bottom for now, but maybe it fits somewhere else better. cc @dannify